web/js/layers/wv.layers.modal.js

```diff
@@ -53,11 +53,8 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
     // Create tiles
     render();
 
-    $addBtn.click(function(e) {
-      $(self.selector).dialog("open");
```

Comment:
Make sure the add layers button still works.

Reply:
I checked it out locally after the merge and it looks to be working fine.